docs/running-in-production/docker-compose.md

```diff
@@ -15,13 +15,13 @@ Then access PrairieLearn from port `3000`.
 PrairieLearn can be configured by a `config.json` in the root of the repository.
 
 - First make the file `config.json` in your root repository.
-- Add the following line to `docker-compose-production.yml` under `volumes`:
+- Add the following line to `docker-compose-production.yml` under `services:pl:volumes`:
```

Comment:
I think this is a more conventional way to describe nested properties in YAML documents:

```suggestion
- Add the following line to `docker-compose-production.yml` under `services.pl.volumes`:
```